hack/make-metrics-doc.sh

```diff
@@ -35,7 +35,7 @@ function get_metrics_url() {
         host_ip=$(kubectl get pod -n kube-system $pod_name -o jsonpath="{.status.hostIP}")
```

Comment:
This line can be removed if `localhost` works fine.

Reply:
@jainpulkit22  have you pushed the latest change?  These two comments are still not addressed.

Reply:
yes i have pushed them just now, was testing few things.